docs/development_setup.md

```diff
@@ -54,8 +54,9 @@ A minimal database setup for running tests is provided. If the build argument `O
 
 ### Requirements
 
-- Python 3.8+
-- Poetry 1.1.0+
+- Python: ≥ 3.8 and ≤ 3.11
+- R: ≥ 4.0
```

Comment:
I don't this this is sufficient, please have a look at the [CI results](https://jenkins.ohsome.org/blue/organizations/jenkins/ohsome-quality-analyst/detail/PR-170/109/pipeline).

Reply:
I think it does. The CI failed because of some kind of `httpx` connection error during the run of `tests/integrationtests/test_indicator_ghs_pop_comparison_roads.py`. Not sure why there needs to be an `httpx` connection to the internet during this test but I will look into this.

Regarding the logging errors at the end of the pytest run (which are related to the rpy2 and pytest packages): There exists an issue to discuss this. #217

Reply:
The avove mentioned integration test is missing the vrc decorator and hence will need to contact the ohsome API. I will make a PR to fix this.